### ISSUE TITLE
Nothing happens, after instalation

### ISSUE THREAD

**Issue by bjnobre** [maharmstone/btrfs#596]
No disk are mounted. I can see on device manager that BTRFS CONTROLLER is present, but nothing more.
How can I get the errors I'm getting?

**Comment by zdenkor** [comment]
The same for me.  Windows 11 22H2, build 22621.2283.

**Comment by ValentineSokol** [comment]
Same issue here.

**Comment by wobblyhen920** [comment]
Same here, unfortunately. Tried both with the .inf installation and the choco one. BTRFS controller and BTRFS volume are visible in Device Manager but there is no way to access them or assign a letter. 

Windows 11 Pro 22H2 22621.2283. No safe boot or TPM. I have implemented the regedit hack nevetheless. I have also checked btrfs.sys on system32/drivers and its size corresponds to the x64 file on the zip.

Please see below some tests on cmd. 

`
C:\Windows\System32>sc query btrfs                                                                                                                                                                                                              NOME_SERVIZIO: btrfs                                                                                                          
TIPO                   : 1  KERNEL_DRIVER                                                                               
STATO                  : 4  RUNNING                                                                                                             
(STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)                                                     
CODICE_USCITA_WIN32    : 0  (0x0)                                                                                       
CODICE_USCITA_SERVIZIO : 0  (0x0)                                                                                       
PUNTO_CONTROLLO          : 0x0                                                                                          
INDICAZIONE_ATTESA     : 0x0                                                                                      

C:\Windows\System32>sc start btrfs                                                                                      
[SC] StartService OPERAZIONI NON RIUSCITE 1056:                                                                                                                                                                                                 Un'istanza del servizio è già in esecuzione.  [in English: an instance of the service is already being executed]                                                                                                                                                                                                                                                                                                                
`

**Comment by maharmstone** [comment]
Thanks @wobblyhen920, that helps. I'll try to reproduce. mountmgr ought to be assigning the volume a drive letter, I'm not sure why it's not doing.

**Comment by maharmstone** [comment]
I've just tried on a fresh install of 22621.2283 and it worked fine.

@wobblyhen920, you haven't installed something like Veracrypt have you? IIRC it used to disable automount without telling you. I don't know if they fixed it.

Can you please check in `diskpart` whether automounting is on or off? (`automount`)

**Comment by wobblyhen920** [comment]
I'm happy to report that WinBtrfs is working fine now. Unfortunately, I'm not able to isolate the cause. Since my last message, I have: 

- uninstalled every Paragon software;
- installed and reinstalled WinBtrfs many times;
- and, more importantly, deleted and re-created the BTRFS partition. In fact, the BTRFS (which I created via [ntfs2btrfs](https://github.com/maharmstone/ntfs2btrfs)) was somewhat broken. I didn't test it enough on Linux before trying to access it in Windows. It went read-only randomly on Linux and showed many errors when using `btrfs check`. Therefore, I used `btrfs restore`, deleted the partition and then re-created it.  I got back on Windows and re-installed WinBtrfs (which I had previously uninstalled) via the inf installation. It didn't work at first. After a few reboots, it worked smoothly (unfortunately, I couldn't say why). 

As of now, automounting is working fine. 
I haven't been using Veracrypt. 

Thank you for your software!
